tests/tga/bug00247.c

```diff
@@ -0,0 +1,14 @@
+#include <stdio.h>
+
+#include "gd.h"
+#include "gdtest.h"
+
+int main(int argc, char **argv)
```

Comment:
Unused `argc` and `argv` is all over places.  I will possibly make some time in future to do such cleanups. (Also more tests are failing with ASAN that probably also needs some attention...)

Thanks for the review, I'll merge it.